app.ts

```diff
@@ -69,8 +68,11 @@ export const tryParseRemoteDataSettings = (
   const remoteDataUserSettings =
     clarinetToml.repl?.["remote_data"] ?? undefined;
 
+  // Default to Hiro API when no URL is provided (matches Clarinet behavior).
+  // This ensures correct deployment plan processing and remote data decisions.
+  const apiUrl = remoteDataUserSettings?.["api_url"] || "https://api.hiro.so";
```

Comment:
Any concerns about rate limits? Otherwise LGTM.

Reply:
Clarinet MXS supports API keys. It'll take the one set in the env var `HIRO_API_KEY` if any